woocommerce/woocommerce
# Add package dependencies so they can be built correctly

### All Submissions:






### Changes proposed in this Pull Request:



Closes 7-gh-woocommerce/platform-private

There was an issue when building `e2e-utils` and `e2e-environment` where files where not being brought in. I don't fully understand how that could have change just by moving folder structures but I am sure there is a simple explanation however since this is an urgent request, I put in a bandaid to include some needed packages to make the build work.

### How to test the changes in this Pull Request:

1. Run `pnpm install` or `pnpm nx build @woocommerce/e2e-utils` and `pnpm nx build @woocommerce/e2e-environment`.
2. Ensure the files for the two packages are built correctly by checking `packages/js/e2e-environment/build/setup/jest.setup.js` and `packages/js/e2e-utils/build/index.js`
3.

### Other information:




### Changelog entry

> Fix build process for e2e-environment and e2e-utils packages.

### FOR PR REVIEWER ONLY:
